tutorials/04-switching-physics-engines.md

```diff
@@ -0,0 +1,99 @@
+\page switchphysicsengines Physics engines
```

Comment:
Since this specifically address switching physics engines in ign-gazebo, should the tutorial be added to ign-gazebo?

Also, I think you also need to add this to `tutorials.md.in` for it to show up in the tutorials page

Reply:
@azeey I derived this tutorial [here](https://github.com/ignitionrobotics/ign-gazebo/blob/master/tutorials/physics.md) :). We agree that there should be a version in `ign-physics` as well.